guides.md

```diff
@@ -253,7 +393,7 @@ This is a checklist for onboarding a new Member to the Co-operative 🚀
   - Community `+hyphacoop:tomesh.net`
 
 1. Set up [`hypha.coop` email](https://link.hypha.coop/email):
-  - Create new mailbox with Mailcow
+  - [Create new mailbox with Mailcow](#creating-new-inboxes-administrators)
   - Set up initial forwarder
```

Comment:
@hyphacoop/ops-wg is this forwarder the initial one within the webmail inbox itself? If so, I think we deal with it in the guide, so could prob remove, or at least rephrase (to convey that it's a summary of things done in guide)

Reply:
>this forwarder the initial one within the webmail inbox itself

I don't understand what the ask is here. Changing the onboard guide? @hyphacoop/operations-wg doesn't know about this.